doc/wg/network/README.md

```diff
@@ -0,0 +1,48 @@
+Network Working Group (NWG)
+=================================
+
+- Working Group Charter
+- Adopted 7/28/2023
+
+## Goals
+
+The goals of the Network Tock Working Group (NWG) are to:
+
+- design a set of interfaces (traits) for networking
+- design the user land interfaces for networking
+- define how buffer management should be implemented
+- determine other kernel infrastructure useful for networking
+
+## Members
+
+- Alexandru Radovici (Chair), OxidOS Automotive
+- Branden Ghena, Northwestern University
+- Leon Schuermann, Princeton University
+- Tyler Potyondy, UCSD
+- Ioan-Cristian Cirstea, OxidOS Automotive
+
+## Membership and Communication
+
+The networking working group membership is a subset of the people who have
+commit (pull request merge) permissions on the Tock repository. It is
+intended to be a smaller group that represents the major perspectives
+and issues, rather than a complete group. Group membership is decided by
+the group: the exact process is not yet determined and may organically
+evolve as the group gains momentum.
+
+The group has a teleconference call once every two weeks. All working group members
+participate in the call. Other people may be invited to participate to
+help contribute to particular topics or on-going discussions. The
+working group chair decides who beyond the working group members may
+participate in the call.
+
+The working group publishes detailed notes of its calls. These will be
```

Comment:
```suggestion

The working group publishes detailed notes of its calls. These will be
```

Reply:
I don't understand what the suggestion is here. The "suggested change" box has the - and + exactly identical. Unless it's that you added a newline??